sql/information_schema/information_schema.go

```diff
@@ -2855,6 +2806,10 @@ func viewsInDatabase(ctx *Context, db Database) ([]ViewDefinition, error) {
 	return views, nil
 }
 
+func remoteBackTicks(s string) string {
```

Comment:
Looks like a typo... probably meant removeBackticks? 😉 

```suggestion
func removeBackticks(s string) string {
```